modules/assisted-installer-assisted-installer-prerequisites.adoc

```diff
@@ -0,0 +1,60 @@
+// This is included in the following assemblies:
+//
+// installing-on-prem-assisted.adoc
+:_content-type: CONCEPT
+
+[id='assisted-installer-prerequisites_{context}']
+= Assisted Installer prerequisites
+
+Successful installation with the {ai-full} requires ensuring the following prerequisites. The installer will validate these prerequisites before installation.
+
+== Hardware
+
+For control plane nodes or the {sno} node, nodes must have at least:
```

Comment:
According to the ISG ([link](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#lead-in-wording)), the lead-in to a list should be a complete sentence. 

This comment applies to the subsequent lists in the module.